Vagrantfile

```diff
@@ -1,9 +1,20 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+unless Vagrant.has_plugin?("vagrant-service-manager")
+  raise <<-MSG
+    vagrant-service-manager plugin is required for projectatomic/adb.
+    Kindly install the plugin: `$ vagrant plugin install vagrant-service-manager`
+  MSG
+end
+
 Vagrant.configure(2) do |config|
+
   config.vm.box = "projectatomic/adb"
 
   config.vm.network "private_network", type: "dhcp"
 
+  # enable multiple providers as comma separated list. Eg: 'docker, openshift'
+  config.servicemanager.providers = 'docker'
```

Comment:
> For docker provider yes, for other providers we may need config in Vagrantfile.

Ok, seems reasonable. We seem to agree the proper configuration of Docker is a given. There is no need to explicitly configure it. What other providers are there? Kubernetes vs OpenShift? Is really OpenShift not a super-set of Kubernetes here? Also OpenShift is the default. It could be generated unless something else is explicitly configured. Again, the aim is zero config _Vagrantfile_. Let's discuss the need for configuring other providers, but Docker should just work.